common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,14 +244,21 @@ class ModuleI18nService(object):
     i18n service.
 
     """
+    log.info('************* ModuleI18nService Usage!!!! **********')
 
     def __getattr__(self, name):
-        return getattr(django.utils.translation, name)
+        msg = '************* ModuleI18nService getattr call!!! Attr: {0} ************'.format(name)
+        log.info(msg)
+        try:
+            return getattr(xblock_translation, name)
+        except:
+            return getattr(django.utils.translation, name)
```

Comment:
Yes, you are correct -- I've got some more proto-code on my local machine that has gone beyond this initial experiment.  Further, I'm now thinking that maybe we need to somehow register the XBlock's locale folder + PO/MO files (or even just the data) with the platform at startup, and then this information could be combined with the base translations.

So rather than 'check xblock, if not check runtime service', it would always be 'check runtime service' and the xblock's translation info would be available through the runtime thanks to the installation process.